code/modules/hydroponics/spreading/spreading.dm

```diff
@@ -67,6 +67,7 @@
 	max_health = round(seed.endurance/2)
 	if(seed.spread == 1)
 		limited_growth = 1
+		plane = CREEPER_LAYER
```

Comment:
>plane = X_LAYER
uhhhhhhhh

Reply:
"layer = " then, I'm guessing? vv wasn't working right on my end to check, but I can test on the test server later.